src/channel.rs

```diff
@@ -93,6 +94,11 @@ pub enum OptTarget {
     Throughput,
 }
 
+pub enum LbPolicy {
+    Default,
```

Comment:
What do you think replace `Default` with `PickFirst`? `PickFirst` is another load balancing policy.

https://github.com/pingcap/grpc/blob/fe6bf3ffe64e3a0e086659abec02897a463cede2/src/core/plugin_registry/grpc_plugin_registry.c#L75

Reply:
Thanks! I've gone with your suggestion here.